metricbeat/docs/modules/gcp.asciidoc

```diff
@@ -143,6 +147,12 @@ Google Cloud Platform offers the https://cloud.google.com/monitoring/api/metrics
 
 If you also want to *extract service labels* (by setting `exclude_labels` to false, which is the default state). You also make a new API check on the corresponding service. Service labels requires a new API call to extract those metrics. In the worst case the number of API calls will be doubled. In the best case, all metrics come from the same GCP entity and 100% of the required information is included in the first API call (which is cached for subsequent calls).
 
+We have updated our field names to align with ECS semantics. As part of this change:
+
+*  `cloud.account.id` will now contain the Google Cloud Organization ID (previously, it contained the project ID).
+*  `cloud.account.name` will now contain the Google Cloud Organization Display Name (previously, it contained the project name).
+*  New fields `cloud.project.id` and `cloud.project.name` will be added to store the actual project ID and project name, respectively.
+
```

Comment:
As @lucabelluccini suggested, could you also explain how to revert to the previous behavior for users who may depend on having the project ID in the `cloud.account.*` fields?

I think adding a processor in the @custom pipeline is good enough.

We need to do the same in the Elastic Agent integrations docs.

Reply:
Something like this?

```json
[
  {
    "set": {
      "field": "cloud.account.id",
      "value": "{{cloud.project.id}}",
      "if": "ctx?.cloud?.project?.id != null"
    }
  },
  {
    "set": {
      "field": "cloud.account.name",
      "value": "{{cloud.project.name}}",
      "if": "ctx?.cloud?.project?.name != null"
    }
  },
  {
    "remove": {
      "field": [
        "cloud.project.id",
        "cloud.project.name"
      ],
      "ignore_missing": true
    }
  }
]
```

Labels with current implementation:

```yaml
    "cloud": {
    "account": {
        "id": "992493199029"
      },
      "project": {
        "id": "elastic-observability",
        "name": "elastic-observability"
      },
    },
```

After pipeline:

```yaml
          "cloud": {
            "project": {},
            "account": {
              "name": "elastic-observability",
              "id": "elastic-observability"
            }
          },
```

Reply:
yep!